.github/workflows/ci.yml

```diff
@@ -6,6 +6,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
```

Comment:
You need a `- uses: actions/setup-node@v3.1.1` line above this `with:`

Reply:
ohhhh dummy